app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+
+import { Formik, Form as ContactForm } from "formik";
```

Comment:
Formik is the entity. So prefix by it rather than some other term.
```suggestion
import { Formik, Form as FormikForm } from "formik";
```